src/components/calculator.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './calculator.css';
+import CalcButton from './button';
+
+function Calculator() {
```

Comment:
- [ ] Your _Calculator_ component _(Calculator.js)_ is not classed based. Kindly change this component to a class-based component.